test/transcript/TranscriptEventTestDataHelper.ts

```diff
@@ -26,7 +27,7 @@ export const TRANSCRIPT_EVENT_TEST_VECTORS = {
     'CisKKQgBENKF2MwEGgt0ZXN0LXJlZ2lvbiISdGVzdC1jb25maWd1cmF0aW9uCisKKQgCENKF2MwEGgt0ZXN0LXJlZ2lvbiISdGVzdC1jb25maWd1cmF0aW9uCisKKQgDENKF2MwEGgt0ZXN0LXJlZ2lvbiISdGVzdC1jb25maWd1cmF0aW9uCisKKQgEENKF2MwEGgt0ZXN0LXJlZ2lvbiISdGVzdC1jb25maWd1cmF0aW9uCisKKQgFENKF2MwEGgt0ZXN0LXJlZ2lvbiISdGVzdC1jb25maWd1cmF0aW9u'
   ),
   TRANSCRIPT_SINGLE: decode(
-    'Cp0BEpoBCpcBCpQBCkcKBFRlc3QQ+eS4xbAvGhNzcGVha2VyLWF0dGVuZGVlLWlkIhhzcGVha2VyLWV4dGVybmFsLXVzZXItaWQoo924xbAvMAE4AQpCCgEuEIvluMWwLxoTc3BlYWtlci1hdHRlbmRlZS1pZCIYc3BlYWtlci1leHRlcm5hbC11c2VyLWlkKIrluMWwLzACEgVUZXN0Lg=='
+    'CpUCEpICCo8CCowCCkwKBFRlc3QQ++uqzMQvGhNzcGVha2VyLWF0dGVuZGVlLWlkIhhzcGVha2VyLWV4dGVybmFsLXVzZXItaWQopeSqzMQvMAE4AUIBMUgBCkIKAS4QjeyqzMQvGhNzcGVha2VyLWF0dGVuZGVlLWlkIhhzcGVha2VyLWV4dGVybmFsLXVzZXItaWQojOyqzMQvMAISBVRlc3QuGjwKA1BJSREAAAAAAADwPxoVQ29udGVudCBpcyBhIFBJSSBkYXRhII3sqszELyiO7KrMxC8yB0FkZHJlc3MaMwoDUElJEQAAAAAAAPA/GhVDb250ZW50IGlzIGEgUElJIGRhdGEgjeyqzMQvKI7sqszELw=='
```

Comment:
Backward compatibility is a key in this PR. I recommend adding more data and tests instead of modifying the existing ones.

Reply:
Sure I can do that. Definitely would be better that way.